mavsdk/tune.py

```diff
@@ -258,9 +258,10 @@ def translate_to_rpc(self, rpcTuneDescription):
             
         rpc_elems_list = []
         for elem in self.song_elements:
-            rpc_elem = tune_pb2.SongElement()
-            elem.translate_to_rpc(rpc_elem)
-            rpc_elems_list.append(rpc_elem)
+                
+            rpc_elem = tune_pb2.SongElement
+            rpc_elems_list.append(elem.translate_to_rpc())
```

Comment:
This doesn't look correct, does it?

Reply:
oh, I don't know why I kept `rpc_elem` at all :see_no_evil:. My bad!

Reply:
Should be better now :+1: